src/jdk.incubator.vector/share/classes/jdk/incubator/vector/VectorOperators.java

```diff
@@ -577,7 +577,7 @@ static boolean opKind(Operator op, int bit) {
     /** Produce saturating unsigned {@code a+b}.  Integral only.
      * @see VectorMath#addSaturatingUnsigned(int, int)
      */
-    public static final Binary SUADD = binary("SUADD", "+", VectorSupport.VECTOR_OP_SUADD, VO_NOFP);
+    public static final Associative SUADD = assoc("SUADD", "+", VectorSupport.VECTOR_OP_SUADD, VO_NOFP+VO_ASSOC);
```

Comment:
```suggestion
    public static final Associative SUADD = assoc("SUADD", "+", VectorSupport.VECTOR_OP_SUADD, VO_NOFP + VO_ASSOC);
```

Reply:
Ordinarily I'd agree, but I'm going to keep with the established pattern in this file.